sample_apps/run_example.sh

```diff
@@ -407,20 +407,7 @@ function emded_policy_in_example_app() {
 # caller cannot pass-in the sample-app-name. So, fall back to using
 # a global variable.
 function compile_app() {
-    case "${SampleAppName}" in
-        "simple_app")
-            compile_simple_app
-            ;;
-
-        "simple_app_under_oe")
-            compile_simple_app_under_oe ""
-            ;;
-
-        *)
-            echo "${Me}: Unknown sample app name '${app_name}'."
-            return
-            ;;
-    esac
+    compile_"${SampleAppName}"
```

Comment:
The pattern of diffs in this change set is exactly of this nature:

- Rip out multiple lines of case / switch lines.
- Replace with single line calling to minion fn whose name is of the form : `<_verb_>_${app_name}`, or something to that effect.

This refactoring reduces the # of lines in this file and makes it easy to drop-in support for newer sample apps.